bentoml/adapters/multi_image_input.py

```diff
@@ -39,61 +50,110 @@ class MultiImageInput(BaseInputAdapter):
         ImportError: imageio package is required to use ImageInput
 
     Example usage:
-        ```
-        class MyService(BentoService):
-            @bentoml.api(input=MultiImageInput(input_names=('imageX', 'imageY')))
-            def predict(self, image_groups):
-                for image_group in image_groups:
-                    image_array_x = image_group['imageX']
-                    image_array_y = image_group['imageY']
-        ```
+
+    >>> class MyService(BentoService):
+    >>> @bentoml.api(input=MultiImageInput(input_names=('imageX', 'imageY')))
```

Comment:
This is great to have a code example in the docstring.  Let's also add accepted_image_formats to this.

Reply:
not many user will need the accepted_image_formats option, I think it's ok to keep it out